meta-client/src/main/java/org/apache/helix/metaclient/api/MetaClientInterface.java

```diff
@@ -36,7 +36,11 @@ enum EntryMode {
 
     // The node will not be automatically deleted when the last sub-entry of the node is deleted.
     // The node is an ephemeral node.
-    CONTAINER
+    CONTAINER,
+
+    // If the entry is not modified within the TTL and has no children it will become a candidate
+    // to be deleted by the server at some point in the future.
```

Comment:
terminology is still very ZK like.  Pure KV doesn't have concept of "child" node. I understand that for ZK, we need to maintain as Helix is using it.